forest/benchmarking/direct_fidelity_estimation.py

```diff
@@ -310,9 +310,12 @@ def acquire_dfe_data(qc: QuantumComputer, expr: TomographyExperiment, n_shots=10
     if mitigate_readout_errors:
         res = list(measure_observables(qc, expr, n_shots=n_shots, active_reset=active_reset))
     else:
-        res = list(measure_observables(qc, expr, n_shots=n_shots, active_reset=active_reset, readout_symmetrize=None,
+        res = list(measure_observables(qc, expr, n_shots=n_shots, active_reset=active_reset, symmetrize_readout=None,
                                        calibrate_readout=None))
 
+    # identify the qubits being measured
+    qubits = list(set(e[0].out_operator for e in expr))
```

Comment:
don't you need  to call `get_qubits()` on `out_operator`?

Reply:
Ah, crud. Fixed it now.

Reply:
But I'm also puzzled why this didn't get picked up by the test. I tested replacing `test_qc` with a `1q-qvm` in which case `assert dfe_estimate.qubits == [0]` fails, so I'm assuming this has something to do with the pytest fixture. I will think about this some more.

Reply:
So it turned out that nothing in that test was actually being tested. I've made the appropriate change to fix that, which also revealed that using `set(e[0].out_operator.get_qubits() for e in expr)` would've raised a `TypeError: unhashable type: 'list'`. I've updated the code to avoid this error, and updated the test accordingly.